test/powershell/Provider/FileSystemProvider.Tests.ps1

```diff
@@ -0,0 +1,49 @@
+Describe "SplitPath Tests (Windows Only)" -tags "CI" {
```

Comment:
>Windows  [](start = 27, length = 8)

It seems like these should just be added to `test/powershell/modules/Microsoft.PowerShell.Management/Split-Path.Tests.ps1`